modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,160 @@
+= Use Studio to Configure a Connector
```

Comment:
Looking at the informal title for #546 PR, are we now in the "Studio Topics" rather than the "Configuration Overview"? Because what follows is task topics, not overview topics.

Reply:
Correct. This is showing a general path for using Studio to configure a connector.